### PR TITLE
feat: change labels for services to be more consistent with workload pods

### LINKED ISSUE #22: Pod and Service labels consistency
It'll be nice to have the same labels for pods and services to able to reuse the selector find services and pods corresponding to an app GUID. Right now pods have `cloudfoundry.org/app_guid` and `cloudfoundry.org/guid` labels and services have `cloudfoundry.org/app`. 

AFAIK, having the same label is a good practice in Kubernetes which eases operators' experience and leverages tools working Kubernetes cluster to group objects under the same entity. https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

### PR DESCRIPTION
**Pull request by mike1808**
fixes #22

### CODE REVIEW

File: cfroutesync/webhook/service_builder.go [hunk 1]
```diff
@@ -45,7 +45,8 @@ func routeToServices(route models.Route, template Template) []Service {
 					}},
 			},
 		}
-		service.ObjectMeta.Labels["cloudfoundry.org/app"] = dest.App.Guid
+		service.ObjectMeta.Labels["cloudfoundry.org/app_guid"] = dest.App.Guid
+		service.ObjectMeta.Labels["cloudfoundry.org/guid"] = dest.App.Guid
```
Comment (tcdowney) [inline cfroutesync/webhook/service_builder.go]: This is actually a `Process` GUID, not an `App` GUID. These will not always be the same value:
```
labels := map[string]string{
		LabelGUID:                        lrp.GUID,
		LabelProcessType:                 lrp.ProcessType,
		LabelVersion:                     lrp.Version,
		LabelAppGUID:                     lrp.AppGUID,
		LabelSourceType:                  appSourceType,
		rootfspatcher.RootfsVersionLabel: m.RootfsVersion,
	}
```

https://github.com/cloudfoundry-incubator/eirini/blob/86bf6bb0408a0b61944e9a849adf02f1af6b6ed4/k8s/statefulset.go#L523-L527

Comment (ndhanushkodi) [reply cfroutesync/webhook/service_builder.go]: the "cloudfoundry.org/guid" label would have the process guid in it. We think we shouldn't need to process guid to select processes, and if we ever want to select a process we could use app guid + process type. we think you can just remove the "cloudfoundry.org/guid" label.

- nitya and tim

**Review by ndhanushkodi — CHANGES_REQUESTED** [review]

File: cfroutesync/webhook/service_builder.go (hunk 1 above)
Comment (ndhanushkodi) [reply cfroutesync/webhook/service_builder.go]: here's an example to show how they can diverge:

1. `cf push dora-health`
1. inspect the labels on its pod with `kubectl`

they will look something like this:
```yaml
labels:
    cloudfoundry.org/app_guid: 80474cf4-0030-46ac-8d79-2ed4ff31fa2e
    cloudfoundry.org/guid: 80474cf4-0030-46ac-8d79-2ed4ff31fa2e
    cloudfoundry.org/process_type: web
    ...
```

1. do a rolling deploy `cf v3-zdt-push dora-health` of the app (default behavior in the v7 cli): `cf v3-restart dora-health`
1. check the labels again
```yaml
labels:
    cloudfoundry.org/app_guid: 80474cf4-0030-46ac-8d79-2ed4ff31fa2e
    cloudfoundry.org/guid: a4b8f920-3ecf-4beb-8b06-d90aefd5c9dc
    cloudfoundry.org/process_type: web
    ...
```

**note** `cf v3-zdt-push` doesn't completely work on our eirini-bosh hybrid environments


* nitya && tim
cc @cwlbraa

**Review by tcdowney — APPROVED** [review]
/lgtm

**Review by ndhanushkodi — APPROVED** [review]

File: cfroutesync/webhook/service_builder.go (hunk 1 above)
Comment (cwlbraa) [reply cfroutesync/webhook/service_builder.go]: sorry i missed this mention, still getting used to the new github ui. eirini doesn't support rolling deploys yet!

**Pull request merged**